pages/api_integration-guides/how_to_transfer.mdx

```diff
@@ -0,0 +1,214 @@
+import { Tab, Tabs } from "nextra-theme-docs";
+
+# How to transfer
```

Comment:
Suggest something more descriptive as a nav item, like "Transfer tokens between accounts" (or just How to transfer tokens)

nit: the nav item is title-cased "How to Transfer." It's not totally consistent but looks like most these pages are not, so I'd make both the header and the nav item sentence-case, "How to transfer"

Reply:
done! I think (`How to transfer -> How to transfer tokens between accounts`)